docs/examples/301/modules-vwan-to-vnet-s2s-with-fw/vnet.bicep

```diff
@@ -0,0 +1,189 @@
+param location string {
+  default: resourceGroup().location
+  metadata: {
+    description: 'Specifies the Azure location where the resource should be created.'
+  }
+}
+param vnetname string {
+  metadata: {
+    description: 'Specifies the name to use for the VNet.'

```

Comment:
descriptions like this don't add much more detail beyond the param name itself and this ends up being noise. Consider getting rid of basic descriptions like this

Reply:
cleaned up the parameters